lib/src/Pos/Util/UserSecret.hs

```diff
@@ -94,14 +94,11 @@ deriving instance Eq EncryptedSecretKey => Eq WalletUserSecret
 
 makeLenses ''WalletUserSecret
 
-fixedNM :: NetworkMagic
-fixedNM = NetworkMainOrStage
-
 instance Buildable WalletUserSecret where
     build WalletUserSecret{..} =
         bprint ("{ root = "%addressF%", set name = "%build%
                 ", wallets = "%pairsF%", accounts = "%pairsF%" }")
-        (makeRootPubKeyAddress fixedNM $ encToPublic _wusRootKey)
+        (makeRootPubKeyAddress NetworkMainOrStage $ encToPublic _wusRootKey)
```

Comment:
Correct me if I am wrong: won't this return a wrong address result for testing networks?

Reply:
Yes, that's correct. This slipped my mind, actually. I may need to add a `NetworkMagic` value to `WalletUserSecret`. However, I wonder if that's a good idea given that this `NetworkMagic` field within `WalletUserSecret` will never be used in any place other than its `Buildable` instance. Will look into either changing the output from `Buildable` or adding this field.

Reply:
Decided to change the `Buildable` instance such that the public key is printed instead of the address.